test/badnets.coffee

```diff
@@ -24,13 +24,14 @@ vows.describe('IPs with bytes greater than 255')
     .addBatch
         '209.256.68.22/255.255.224.0': shouldFailWithError 'Invalid net'
         '209.180.68.22/256.255.224.0': shouldFailWithError 'Invalid mask'
-        '1.1.1.1/': shouldFailWithError 'Invalid mask'
+#        '1.1.1.1/': shouldFailWithError 'Invalid mask'      # converted to /32, watch proposed fix in lib/netmask.coffee
```

Comment:
I'm more for being strict on what we accept and not trying to guess what is the sh** coming in :)